docs/tutorial2023_unfolding/unfolding_exercise.md

```diff
@@ -91,47 +90,49 @@ where you can see the contributions from various background processes, namely Z+
 
-While the uncertainties on the POI can be extracted in multiple ways, the most robust way is to run the likelihood scans for POI corresponding to each gen-level bin, it allows to spot discontinuities in the likelihood shape in case of the problems with the model. 
+While the uncertainties on the POI can be extracted in multiple ways, the most robust way is to run the likelihood scans for a POI corresponding to each gen-level bin, it allows to spot discontinuities in the likelihood shape in case of problems with the model. 
 
 ```shell
-combineTool.py -M MultiDimFit --datacard ws_counting.root -t -1 --setParameters r_zh_250_400=1,r_zh_150_250noj=1,r_zh_75_150=1,r_zh_150_250wj=1,r_zh_gt400=1 --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400 --saveWorkspace --algo=grid --points=100 -P r_zh_75_150 --floatOtherPOIs=1 -n scan_r_zh_75_150
+combineTool.py -M MultiDimFit --datacard ws_counting.root -t -1 --setParameters r_zh_250_400=1,r_zh_150_250noj=1,r_zh_75_150=1,r_zh_150_250wj=1,r_zh_gt400=1 --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400 --algo=grid --points=100 -P r_zh_75_150 --floatOtherPOIs=1 -n scan_r_zh_75_150
 
 ```
 Now we can plot the likelihood scan and extract the expected intervals.
 
 ```shell
-python scripts/plot1DScan.py higgsCombinescan_r_zh_75_150.MultiDimFit.mH120.root -o r_zh_75_150 
+python scripts/plot1DScan.py higgsCombinescan_r_zh_75_150.MultiDimFit.mH120.root -o r_zh_75_150 --POI r_zh_75_150
 ```
-> Repeat for all POIs
+* Repeat for all POIs
 
 ## Shape analysis with control regions
 
 One of the advantages of the maximum likelihood unfolding is the flexibility to choose the analysis observable and include more information on the event kinematics, consequently improving the analysis sensitivity. This analysis benefits from the shape information of the DNN output trained to differentiate the VH(bb) signal from the SM backgrounds. 
 
 The datacards for this part of the exercise located `full_model_datacards/`, where you can find a separate datacard for each region within `full_model_datacards/regions` directory and also a combined datacard `full_model_datacards/comb_full_model.txt`. 
  
-As you will find the datacards also contain several background processes. To control them properly we will add the regions enriched in the respective backgrounds. Then we can define a common set rate parameters for signal and control regions to scale the rates or other parameters affecting their shape.  
+As you will find the datacards also contain several background processes. To control them properly we will add the regions enriched in the respective backgrounds. Then we can define a common set of rate parameters for signal and control regions to scale the rates or other parameters affecting their shape.  
 
 For the shape datacards one has to specify the mapping of histograms and channels/processes as given described below:
```

Comment:
we could also link to the part of the documentation that describes this, for further information.